### PR TITLE
Add cosin

### PR DESCRIPTION
**Pull request by yashrajgupta**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,31 +0,0 @@
-# This workflow will build a Harmony OS project with Gradle and analyse the code using Sonar cloud.
-
-name: Build
-
-on:
-  push:
-    branches: [ master ]
```
Comment (samuel-abhishek) [inline .github/workflows/main.yml]: This is file is required. Please add it

File: LICENSE [hunk 2]
```diff
@@ -1,21 +0,0 @@
...
-
-Copyright (c) 2018 NikitaGordia
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
```
Comment (samuel-abhishek) [inline LICENSE]: THe LICENSE file is required. Please add this again

File: build.gradle [hunk 3]
```diff
@@ -1,27 +1,66 @@
...
-task clean(type: Delete) {
-    delete rootProject.buildDir
+checkstyle {
+    toolVersion "8.43"
 }
+sonarqube {
+    properties {
+        property "sonar.projectKey", "applibgroup_Cosin"
+        property "sonar.organization", "applibgroup"
+        property "sonar.host.url", "http://localhost:9000"
```
Comment (samuel-abhishek) [inline build.gradle]: sonar settings are incorrect. Please rectify

File: gradle.properties [hunk 4]
```diff
@@ -1,17 +1,12 @@
...
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx1536m
-
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
+# If the Chinese output is garbled, please configure the following parameter.
+# org.gradle.jvmargs=-Dfile.encoding=GBK
+systemProp.sonar.host.url=http://localhost:9000
```
Comment (samuel-abhishek) [inline gradle.properties]: delete this info. Not required for the checkin

File: cosin/src/main/java/com/nikitagordia/cosin/Cosin.java [hunk 5]
```diff
@@ -1,56 +1,75 @@
...
+import ohos.agp.text.Font;
+import ohos.agp.utils.Color;
+import ohos.app.Context;
+import com.nikitagordia.cosin.colorAdapters.DefaultColoradapterGB;
 import com.nikitagordia.cosin.textAdapters.DefaultBinaryTextAdapter;
 
-
 /**
- * Created by nikitagordia on 04.03.18.
+ * Created by nikitagordia on 05.03.18.
```
Comment (samuel-abhishek) [inline cosin/src/main/java/com/nikitagordia/cosin/Cosin.java]: Please retain the same java doc as in base library

File: entry/src/main/java/com/nikitagordia/cosinloading/MainAbility.java [hunk 6]
```diff
@@ -0,0 +1,16 @@
+package com.nikitagordia.cosinloading;
```
Comment (samuel-abhishek) [inline entry/src/main/java/com/nikitagordia/cosinloading/MainAbility.java:17]: please add the copyright for this file

File: entry/src/main/java/com/nikitagordia/cosinloading/MyApplication.java [hunk 7]
```diff
@@ -0,0 +1,13 @@
+package com.nikitagordia.cosinloading;
```
Comment (samuel-abhishek) [inline entry/src/main/java/com/nikitagordia/cosinloading/MyApplication.java:17]: please add the copyright for this file

File: entry/src/ohosTest/java/com/nikitagordia/cosinloading/CosinTest.java [hunk 8]
```diff
@@ -0,0 +1,168 @@
+package com.nikitagordia.cosinloading;
```
Comment (samuel-abhishek) [inline entry/src/ohosTest/java/com/nikitagordia/cosinloading/CosinTest.java:17]: Please add the copyright for this file

File: LICENSE (hunk 2 above)
Comment (yashrajgupta) [reply LICENSE]: Added

File: gradle.properties (hunk 4 above)
Comment (yashrajgupta) [reply gradle.properties]: Done

File: entry/src/main/java/com/nikitagordia/cosinloading/MainAbility.java (hunk 6 above)
Comment (yashrajgupta) [reply entry/src/main/java/com/nikitagordia/cosinloading/MainAbility.java:17]: Done

File: build.gradle (hunk 3 above)
Comment (yashrajgupta) [reply build.gradle]: Done

File: cosin/src/main/java/com/nikitagordia/cosin/Cosin.java (hunk 5 above)
Comment (yashrajgupta) [reply cosin/src/main/java/com/nikitagordia/cosin/Cosin.java]: Done

File: entry/src/ohosTest/java/com/nikitagordia/cosinloading/CosinTest.java (hunk 8 above)
Comment (yashrajgupta) [reply entry/src/ohosTest/java/com/nikitagordia/cosinloading/CosinTest.java:17]: Done

**Pull request merged**